index.html

```diff
@@ -4480,15 +4478,28 @@ <h2 id="AudioBufferSourceNode">
           </dt>
           <dd>
             <p>
-              Schedules a sound to stop playback at an exact time. If
-              <code>stop</code> is called again after already having been
-              called, the last invocation will be the only one applied; stop
-              times set by previous calls will not be applied, unless the
-              buffer has already stopped prior to any subsequent calls. If the
-              buffer has already stopped, further calls to <code>stop</code>
-              will have no effect. If a stop time is reached prior to the
-              scheduled start time, the sound will not play.
+              Schedules a sound to stop playback at an exact time, overriding
```

Comment:
Isn't "overriding any previous...stop time." a duplication of the next sentence?